frontend/src/constants/tour-steps.tsx

```diff
@@ -0,0 +1,35 @@
+import type { StepType } from '@reactour/tour';
+
+const TOUR_STEPS: StepType[] = [
```

Comment:
Here you'll also need to coordinate with Álvaro because we'll need to translate these strings.